wiki/Welcome/de.md

```diff
@@ -1,130 +1,117 @@
+![Freundesliste](DE-Friend-list.jpg "Freundesliste")
 
-Beatmapping (Edit)
-==================
+Dies waren die Grundlagen, scheue dich nicht davor nach Hilfe im Spiel oder im Forum zu bitten, das osu!wiki sollen dir auch beim Verständnis des Spieles behilflich sein.
 
-*Vollständige Erklärung: [Beatmapping](DE:Beatmapping "wikilink")*
+## Beatmapping (Edit)
 
-osu! bietet für jeden Spieler die Möglichkeit an seine eigenen Beatmaps zu erstellen. Wenn Sie denken, dass Sie kreativ sind und keine Angst vor dem Lernen haben, dann sollten Sie vielleicht damit beginnen Ihre eigenen Maps zu Liedern zu kreieren. Um zu beginnen, starten Sie einfach den "Edit" Modus und ziehen Sie direkt das Lied im .mp3 Format in osu! Danach sollte Ihnen eine leere Map ohne Namen angezeigt werden. Wie auch immer, Beatmapping ist ein "sehr" komplizierte Prozess, welches ein gewisses Maß an Einsatz fordert. Der beste Weg es zu lernen ist immernoch es selbst auszuprobieren. Sie können sich auch diverse Anleitungen über das Mapping [hier](http://osu.ppy.sh/forum/viewtopic.php?f=56&t=60228) durchlesen. Der "Edit" Modus ist auch ziemlich nützlich, um einen genaueren Blick auf die Map zu werfen bzw. schon vorhandene Beatmaps mit Ihren selbsterstellten Beatmaps zu vergleichen. Sie sollten es auch **vermeiden gerankte Maps zu editieren**, da sie sonst wieder in den **unranked** zurückverfallen (natürlich nur lokal). Sie werden dann auch nicht mehr in der Lage sein einen Score zu submitten bzw. pp zu erhalten, solange die Map nicht wieder in ihren gerankten Zustand zurückgekehrt ist.
+*Vollständige Erklärung: [Beatmapping](/wiki/Beatmapping/ "Beatmapping")*
 
-Nützliche Hilfsmittel
-=====================
+*osu!* bietet für jeden Spieler die Möglichkeit an seine eigenen Beatmaps zu erstellen. Wenn du denkst, dass du kreativ bist und keine Angst vor dem Lernen hast, dann solltest du vielleicht damit beginnen deine eigenen Maps zu Liedern zu kreieren. Um zu beginnen, starte einfach den `Edit` Modus und ziehen das Lied direkt im `.mp3` Format in *osu!* Danach sollte eine leere Map ohne Namen angezeigt werden. Wie auch immer, Beatmapping ist ein **sehr** komplizierter Prozess, welches ein gewisses Maß an Einsatz fordert. Der beste Weg es zu lernen ist immernoch es selbst auszuprobieren. Du kannst dir auch diverse Anleitungen über das Mapping [hier](http://osu.ppy.sh/forum/viewtopic.php?f=56&t=60228) durchlesen. Der `Edit` Modus ist auch ziemlich nützlich, um einen genaueren Blick auf die Map zu werfen bzw. schon vorhandene Beatmaps mit deinen selbsterstellten Beatmaps zu vergleichen. Du solltest es auch **vermeiden gerankte Maps zu editieren**, da sie sonst wieder in den **Unranked** zurückverfallen (natürlich nur lokal). Du wirst dann auch nicht mehr in der Lage sein einen Score zu submitten bzw. PP zu erhalten, solange die Map nicht wieder in ihrem gerankten Zustand zurückgekehrt ist.
```

Comment:
use the shorter link
`http://osu.ppy.sh/forum/viewtopic.php?f=56&t=60228` -> `https://osu.ppy.sh/forum/t/60228`